test/check-machines-create

```diff
@@ -778,6 +791,29 @@ class TestMachinesCreate(VirtualMachinesCase):
             virt_install_cmd_out = self.machine.execute(virt_install_cmd)
             self.assertIn("--cloud-init user-data=", virt_install_cmd_out)
 
+        def createAndVerifyVirtInstallArgsQemuConf(self):
+            self.browser.click(".pf-c-modal-box__footer button:contains(Create)")
+            self.browser.wait_not_present("#create-vm-dialog")
+            # Get IP address of 'spice_listen = "IP_ADDRESS"' from qemu.conf, and trim quotes and endline
+            spice_ip = self.machine.execute("cat /etc/libvirt/qemu.conf | grep '^spice_listen' | awk '{print $3}' | sed 's/\"//g' | tr -d '\\n'")
```

Comment:
Ugh, five different shell tools... Replace the first four with `awk '/^spice_listen/ {sub(/"/, ""); print $3}', and replace the `tr` with a Python `execute(...).strip()`.

But it would IMHO be even better to just pass the spice_listen and vnc_listen values as parameters to createAndVerifyVirtInstallArgsQemuConf() -- then it can write qemu.conf itself, and does not have to parse them back out of the file that the test just wrote. This is a whole lot simpler and more robust.